input/data/2008/2008ApJ...679.1427A/tev-000014-sed.ecsv

```diff
@@ -0,0 +1,24 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - {name: e_min, unit: TeV, datatype: float64}
```

Comment:
Well, to me whatever we do is all pretty equally bad: at some point we're inventing an `e_ref` at which to put the published `dnde`. I think if I were to do it, I would add it here in the dataset and put a caveat as a comment that this `e_ref` was made up, not published.

If you put it in the tools (e.g. Gammapy SED reader) you're just encouraging bad behaviour to publish a measurement in a bad way. (And it is rare, I haven't seen that before and we have SEDs from > 100 papers by now).